deploy/charts/cert-manager/README.template.md

```diff
@@ -148,6 +151,9 @@ The following table lists the configurable parameters of the cert-manager chart
 | `cainjector.podAnnotations` | Annotations to add to the cainjector pods | `{}` |
 | `cainjector.deploymentAnnotations` | Annotations to add to the cainjector deployment | `{}` |
 | `cainjector.extraArgs` | Optional flags for cert-manager cainjector component | `[]` |
+| `cainjector.serviceAccount.create` | If `true`, create a new service account for the cainjector component | `true` |
+| `cainjector.serviceAccount.name` | Service account to be used for the cainjector component If not set and `cainjector.serviceAccount.create` is `true`, a name is generated using the fullname template |  |
```

Comment:
This is missing `to be used` (which is included in the description for `webhook.name`

Reply:
fixed